metadata/shakyground.json

```diff
@@ -13,6 +13,30 @@
             "useAs": "commandLineArgument",
             "type": "quakeml",
             "schema": "http://quakeml.org/xmlns/quakeml/1.2/QuakeML-1.2.xsd"
+        },
+        {
+            "title": "gmpe",
```

Comment:
This was also in another pull request, but there should be no conflicts I guess...